Formula/g/glow.rb

```diff
@@ -4,6 +4,7 @@ class Glow < Formula
   url "https://github.com/charmbracelet/glow/archive/refs/tags/v1.5.1.tar.gz"
   sha256 "b4ecf269b7c6447e19591b1d23f398ef2b38a6a75be68458390b42d3efc44b92"
   license "MIT"
+  revision 1
```

Comment:
No need to force an update for all users, they can manually reinstall or get it on the next update.